src/main/java/com/springdata/service/PersonServiceImpl.java

```diff
@@ -0,0 +1,50 @@
+package com.springdata.service;
+
+import com.springdata.model.PersonEntity;
+import com.springdata.repository.PersonRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@AllArgsConstructor(onConstructor = @__(@Autowired))
+public class PersonServiceImpl implements PersonService {
+    private PersonRepository repository;
+
+    @Override
+    public void createPerson(PersonEntity person) {
+        repository.save(person);
+    }
+
+    @Override
+    public void updatePerson(PersonEntity persin) {
+        repository.save(persin);
+    }
+
+    @Override
+    public void deletePerson(Long id) {
+        repository.deleteById(id);
+    }
+
+    @Override
+    public PersonEntity findPersonById(Long id) {
+        if (repository.findById(id).isPresent()) {
```

Comment:
It could be rewritten  it in a more elegant way
`repository.findById(id).orElseThrow(new::IllegalArgumentException) `
if you want to throw an exception. But better to use some custom exception
UserNotFoundException for example
or  return null if there is no user 
`repository.findById(id).orElse(null)`